### PR TITLE
Feature/mx 1557 transform grippeweb resources

### PR DESCRIPTION
**Pull request by erichesse**
# PR Context
- linkage to confluence-vvt extractor blocked by MX-1583

# Added
- transform grippeweb resources

### CODE REVIEW

**Review by mr-kamran-ali — COMMENTED** [review]
Great work Eric, dont have any major change requests, just a few little clarifications. Other than the "KeyError" at `mex\grippeweb\transform.py", line 213` things look great.

File: mex/grippeweb/extract.py [hunk 1]
```diff
@@ -14,3 +19,74 @@ def extract_columns_by_table_and_column_name() -> dict[str, dict[str, list[Any]]
         table_name: connection.parse_columns_by_column_name(table_name)
         for table_name in QUERY_BY_TABLE_NAME.keys()
     }
+
+
+def extract_ldap_actors(
```
[the remark this answers is not in the document]
Comment (erichesse) [reply mex/grippeweb/extract.py]: clarified in https://github.com/robert-koch-institut/mex-extractors/pull/57/commits/80fea82c85f0182fa2c232f518c8f28a45f0f368

File: mex/grippeweb/transform.py [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+    """
+    resource_dict = {}
+    mex_persons_by_name = {
+        person.fullName[0]: person for person in extracted_mex_persons_grippeweb
+    }
+    """
+    TODO: (blocked by MX-1583) confluence_vvt_by_identifier_in_primary_source = {
+         source.identifierInPrimarySource: source.stableTargetId
+         for source in extracted_confluence_vvt_sources
+     }"""
```
[the remark this answers is not in the document]
Comment (erichesse) [reply mex/grippeweb/transform.py]: https://github.com/robert-koch-institut/mex-extractors/pull/57/commits/ca04da58ce664e274d7873cc5cbce3552ed2b1b2

File: tests/grippeweb/mocked_grippeweb.py
[the remark this answers is not in the document]
Comment (erichesse) [reply tests/grippeweb/mocked_grippeweb.py:1]: see https://github.com/robert-koch-institut/mex-extractors/pull/69

**Comment by erichesse** [comment]
> Great work Eric, dont have any major change requests, just a few little clarifications. Other than the "KeyError" at `mex\grippeweb\transform.py", line 213` things look great.

fixed in https://github.com/robert-koch-institut/mex-extractors/pull/57/commits/52d3ac579499217919f2dd729c36f228367efb19

**Review by mr-kamran-ali — APPROVED** [review]

**Pull request merged**